tool/validate_firebase.py

```diff
@@ -64,7 +52,7 @@ def usage():
     usage()
     exit(1)
 
-init(crypto_token_path)
+firebase_util.init(crypto_token_path, log)
```

Comment:
accidentally left over from the previous version? I think this should now be:
`firebase_client = firebase_util.init(crypto_token_path, log)`
and `firebase_client = None` should be added back in at the top of the file

Reply:
Refactored